packages/docs/src/pages/products/index.md

```diff
@@ -61,6 +61,10 @@ TypeScript SDK that makes the FHIR spec easier to manage. Includes all FHIR type
 
 Build apps quickly with a ready to integrate react component library. Enable a custom experience for both customers and patients. [Learn More](/docs/tutorials/ui-components)
 
+### Search
```

Comment:
This seems more like an enabling feature than a product on its own. I think it should be removed from this page.

Reply:
Good point, removing